release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -805,6 +805,14 @@ Red Hat does not guarantee backward compatibility for metrics, recording rules,
 
 * You can add and configure additional external Alertmanagers for both platform and user-defined project monitoring stacks.
 * You can disable the local Alertmanager instance.
+* You can assign the new `monitoring-alertmanager-edit` user role to allow non-administrator users to create and silence alerts for default platform monitoring.
++
+[IMPORTANT]
+====
+With this release, the `cluster-monitoring-view` role is now restricted to view-only access for Alertmanager.
```

Comment:
users with only the `cluster-monitoring-view` clusterrole don't have access anymore to the Alertmanager UI at all. They need to be granted the `monitoring-alertmanager-edit` role too.

Reply:
@simonpasquier If I'm understanding correctly, there's no way for a non-admin to get view-only access to the Alertmanager UI, correct?

Reply:
I updated the wording to reflect that a non-admin user needs both roles to be able to create and silence alerts and that there is not a "view-only" role for a non-admin user.